CMakeLists.txt

```diff
@@ -443,6 +445,11 @@ if(WITH_CORE)
     endif()
   endif()
 
+  if (WITH_DRACO)
+    find_package(draco REQUIRED)
```

Comment:
FWIW, this doesn't work for me on Fedora:
```
CMake Error at /usr/share/cmake/draco/draco-targets.cmake:89 (message):
  The imported target "draco::draco_static" references the file

     "/usr/lib64/libdraco.a"
```

That's why I resorted to the FindDraco inclusion in https://github.com/qgis/QGIS/pull/54094